src/coreclr/jit/async.h

```diff
@@ -32,6 +32,7 @@ class Async2Transformation
     unsigned                      m_exceptionVar            = BAD_VAR_NUM;
     BasicBlock*                   m_lastSuspensionBB        = nullptr;
     BasicBlock*                   m_lastResumptionBB        = nullptr;
+    BasicBlock* m_sharedReturnBB                            = nullptr;
```

Comment:
Formatting is a bit off here, but the JIT formatter makes a bunch of changes in various code, so I want to submit those formatting changes in a separate PR.